workspace.go

```diff
@@ -203,6 +203,14 @@ func (w *Workspace) OpenFile(absFilepath, text string) error {
 	w.Loader.done = false
 	w.Loader.stateChange <- absPath
 
+	asc := flattenAscendants(n)
```

Comment:
Scan through other function to see if anything else needs to process the ascendants.

Reply:
Fixed ReplaceFile as well.